publishing/downloadService/get_public_filter_download_test.go

```diff
@@ -0,0 +1,92 @@
+package downloadService
+
+import (
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/ONSdigital/dp-api-tests/testDataSetup/mongo"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/gavv/httpexpect"
+	"github.com/satori/go.uuid"
+	. "github.com/smartystreets/goconvey/convey"
+	"github.com/ONSdigital/dp-api-tests/web/filterAPI"
+	"gopkg.in/mgo.v2"
+)
+
+func TestRedirectToPublicFilterDownload(t *testing.T) {
+
+	filterID := uuid.NewV4().String()
+	filterBlueprintID := uuid.NewV4().String()
+	filterOutputID := uuid.NewV4().String()
+	datasetID := uuid.NewV4().String()
+	instanceID := uuid.NewV4().String()
+	edition := "2017"
+	version := 1
+
+	filterBlueprintDoc := &mongo.Doc{
+		Database:   cfg.MongoFiltersDB,
+		Collection: "filters",
+		Key:        "_id",
+		Value:      filterID,
+		Update:     filterAPI.GetValidFilterWithMultipleDimensionsBSON(cfg.FilterAPIURL, filterID, instanceID, datasetID, edition, filterBlueprintID, version, true),
```

Comment:
minor inconsistency: this uses `true` where other tests use `isPublished`

personally, I'd have two globals like
```
const (
  publishedTrue = true
  publishedFalse = false
)
```
and use those in the function calls